ls1intum/ArTEMiS
# migrate programming exercise repo url and build plan id to participations

### Checklist

### Motivation and Context
In the future we want to show the instructor how many test cases failed in the template project and how many passed in the solution project. 

### Description
Migrate baseRepositoryUrl and baseBuildPlanId to templateParticipation and solutionRepositoryUrl and solutionBuildPlanId to solutionParticipation.

**Please note:**
>Testing on the test server should only be done when the changes have been integrated into develop, because after the data migration, the programming exercises won't have certain attributes any more in the database which might lead to errors when version without these changes are deployed.

### Steps for Testing
1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Navigate to Programming Exercises
4. Check if you still can see the repo url and build plan ids
